frontend/src/metabase/dashboard/components/DashboardGrid.jsx

```diff
@@ -253,7 +239,16 @@ class DashboardGrid extends Component {
   };
 
   onDashCardRemove(dc) {
-    this.setState({ removeModalDashCard: dc });
+    this.props.removeCardFromDashboard({
+      dashcardId: dc.id,
+    });
+    this.props.addUndo({
+      message: t`Removed card`,
```

Comment:
Adding a removed card name may be helpful. @mazameli what do you think?

Reply:
Yeah I can add that, it should be a simple change